apps/qolsysgw/mqtt/updater.py

```diff
@@ -444,6 +445,7 @@ class MqttWrapperQolsysSensor(MqttWrapper):
         QolsysSensorAuxiliaryPendant: 'safety',
         QolsysSensorSiren: 'safety',
         QolsysSensorKeyFob: 'safety',
+        QolsysSensorTemperature: 'heat',
```

Comment:
I guess temperature sensors in Qolsys are actually heat sensors? Or can they trigger for cold temperatures also ?

Reply:
It would of course be cool to create both a cold and heat sensor in HA, but I think that's unfortunately beyond my Python beginner skills.

Reply:
Yeah, I don't think there's a good sensor type that'd work for any of those in HA unfortunately. And having both sensors wouldn't necessarily help since we have only a single sensor.